### ISSUE TITLE
staticbin + http.Filesystem + StaticFS()

### ISSUE THREAD

**Issue by manucorporat** [olebedev/staticbin#2]
If staticbin implementes the http://golang.org/pkg/net/http/#FileSystem interface it could be "mounted" in Gin using the new `StaticFS(relativePath, filesystem, list)`
https://github.com/gin-gonic/gin/blob/develop/routergroup.go#L121-L135

``` go
router := gin.Default()
router.StaticFS("/statc/", staticbin_instance, false)
router.Run(":8080")
```

**Comment by manucorporat** [comment]
I may implement it when gin v1. is released

**Comment by olebedev** [comment]
@manucorporat sure, it's good idea. But now i don't have time to implement is soon. A lot of work..
BTW, what is release date of v1 Gin? :)

**Comment by manucorporat** [comment]
@olebedev  I released Gin v1.0rc1 today:
https://github.com/gin-gonic/gin/releases/tag/v1.0rc1

**Comment by olebedev** [comment]
@manucorporat I have just found http://golang.org/pkg/net/http/#FileSystem imlementation for `go-bindata`! Here it is http://godoc.org/github.com/elazarl/go-bindata-assetfs.

It means that there is no need for `staticbin` package anymore. (
